lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -715,16 +714,18 @@ mkUnsignedTransaction networkId stakeCred ctx selection = do
 _decodeSealedTx
     :: AnyCardanoEra
     -> WitnessCountCtx
-    -> SealedTx ->
-        ( Tx
+    -> SealedTx
+    ->  ( Tx
         , TokenMapWithScripts
         , TokenMapWithScripts
         , [Certificate]
         , Maybe ValidityIntervalExplicit
         , WitnessCount
         )
```

Comment:
I suspect a record type might be more appropriate here, but I'll leave that for another PR.